sig-ops/README.md

```diff
@@ -0,0 +1,27 @@
+# SIG: Ops
+
+This Special Interest Group (SIG) main goal is to coordinate Operations (Ops) activities and best practices across Spinnaker Operators.
+* Production Readiness
+* Security Requirements
+* Reference Architecture(s)  
+
+## Meetings
+
+* Regular SIG Meeting
+  * Every other TBD
+  * [Agenda]
```

Comment:
Can this be linked to an appropriate document?

Reply:
indeed will do